aws-creds/iam.tf

```diff
@@ -59,7 +66,64 @@
 
 #data "aws_caller_identity" "current" {}
 
+resource "aws_iam_role" "terraform_architect_iam_role" {
+  name = "terraform-architect"
+  path = "/"
+  assume_role_policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "AWS": "arn:aws:iam::162808325377:root"
```

Comment:
Should do something like [this](https://github.com/yuvipanda/terraform-deploy/blob/fix/no-users/aws/iam.tf#L36) instead of hardcoding the account id

Reply:
got it fixed, accidentally left that in whoops